### PR TITLE
update FAQ with two questions

### PR DESCRIPTION
**Pull request by curiouskiwi**
Added questions about "less vs more" and Verification deadlines.

### CODE REVIEW

File: cs50x/faqs.adoc [hunk 1]
```diff
@@ -33,6 +33,22 @@ No, afraid CS50x is too large, but you can still start a https://courses.edx.org
 
 '''
 
+[[difference-between-less-comfortable-and-more-comfortable-problems]]
+== What's the difference between Less Comfortable and More Comfortable problems? Do I have to do both?
+
+The "Less Comfortable" are what you might consider the "standard" version of the problem, designed for students who have little or no prior experience. The "More Comfortable" are the "challenge" version, designed for students who consider themselves more comfortable due to prior study/experience before this class.  As such, they may require more concepts than have been covered in the course so far.  
+
+You don't get any extra points for doing the "More" problems. If you submit both, the one with the highest grade will count towards your overall pset grade.  The problem set document will give you the options.
```
Comment (dmalan) [inline cs50x/faqs.adoc]: What does this last sentence mean? :)

File: cs50x/faqs.adoc [hunk 2]
```diff
@@ -33,6 +33,22 @@ No, afraid CS50x is too large, but you can still start a https://courses.edx.org
...
+
+'''
+
+
+[[do-I-have-to-choose-verification-before-starting-the-course]]
+== Do I have to choose Verification before starting the course?
+No!  If you would like an edX Verified Certificate, you can pay for verification at any point in your studies, even after completing all of the coursework.
```
Comment (dmalan) [inline cs50x/faqs.adoc]: Let's change to the gentler "Nope!" And tweak to "... you can pay for verification at any point during the year, even ..."

File: cs50x/faqs.adoc (hunk 1 above)
Comment (curiouskiwi) [inline cs50x/faqs.adoc]: I copied one of my older reddit answers and didn't mean to include that last bit. (it was answering another question!).  Removed.

File: cs50x/faqs.adoc (hunk 2 above)
Comment (curiouskiwi) [inline cs50x/faqs.adoc]: done.

File: cs50x/faqs.adoc [hunk 3]
```diff
@@ -38,14 +38,14 @@ No, afraid CS50x is too large, but you can still start a https://courses.edx.org
 
 The "Less Comfortable" are what you might consider the "standard" version of the problem, designed for students who have little or no prior experience. The "More Comfortable" are the "challenge" version, designed for students who consider themselves more comfortable due to prior study/experience before this class.  As such, they may require more concepts than have been covered in the course so far.
```
Comment (dlloyd09) [inline cs50x/faqs.adoc]: Mind revising to "less comfortable" and "more comfortable," without the capitalization, which is consistent with how they are presented in the problem specs?

File: cs50x/faqs.adoc [hunk 4]
```diff
@@ -38,14 +38,14 @@ No, afraid CS50x is too large, but you can still start a https://courses.edx.org
 
 The "Less Comfortable" are what you might consider the "standard" version of the problem, designed for students who have little or no prior experience. The "More Comfortable" are the "challenge" version, designed for students who consider themselves more comfortable due to prior study/experience before this class.  As such, they may require more concepts than have been covered in the course so far.  
 
-You don't get any extra points for doing the "More" problems. If you submit both, the one with the highest grade will count towards your overall pset grade.  The problem set document will give you the options.
+You don't get any extra points for doing the "More" problems. If you submit both, the one with the highest grade will count towards your overall pset grade.
```
Comment (dlloyd09) [inline cs50x/faqs.adoc]: Similarly here, let's use "more comfortable."

**Pull request merged**